### PR TITLE
add dev_url [ci skip]

### PR DESCRIPTION
**Pull request by carlodri**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: e38627e66ca787f56ad2898699e07cb7ae2049a7dc075d535367cd882c417b9a
 
 build:
-  number: 0
+  number: 2
```
Comment (jakirkham) [inline recipe/meta.yaml]: For metadata changes, we can skip the build number bump.

**Comment by jakirkham** [comment]
Some metadata changes need a re-rendering as well to update the feedstock README for instance. I don’t think this one does, but it is easy to check.

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

**Comment by carlodri** [comment]
@jakirkham no answer from conda-forge-admin means rerender is not necessary?

**Comment by jakirkham** [comment]
Correct.

**Comment by jakirkham** [comment]
Thanks @carlodri. Feel free to merge. :)

**Comment by jakirkham** [comment]
Thanks again @carlodri.

> no answer from conda-forge-admin means rerender is not necessary?

Thanks for mentioning this. The bot should say something. Raised issue ( https://github.com/conda-forge/conda-forge-webservices/issues/176 ) on this point.

**Comment by carlodri** [comment]
you're welcome, happy to contribute where I can!

**Pull request merged**